src/lib/traitify-client.js

```diff
@@ -23,6 +23,29 @@ export default class TraitifyClient {
     this.version = version;
     return this;
   }
+  handlePromise = (requestType, xhr, params) => {
+    const async = xhr;
```

Comment:
Not sure if you want to use `async` here. Might conflict as a reserved word

Reply:
This was a due linting error - no param reassign - I believe it was. I updated the naming and two references. Good catch